lib/cadet/assessments/assessments.ex

```diff
@@ -422,7 +423,7 @@ defmodule Cadet.Assessments do
 
   @spec all_submissions_by_grader(%User{}) ::
           {:ok, [%Submission{}]} | {:error, {:unauthorized, String.t()}}
-  def all_submissions_by_grader(grader = %User{role: role}) do
+  def all_submissions_by_grader(grader = %User{role: role}, group_only \\ false) do
```

Comment:
Can add a `@doc` documenting this function, and explaining `group_only` param

Reply:
Done!